ding_content.make

```diff
@@ -48,8 +48,12 @@ projects[views][version] = "3.8"
 projects[workbench][subdir] = "contrib"
 projects[workbench][version] = "1.2"
 
+projects[wysiwyg][type] = "module"
 projects[wysiwyg][subdir] = "contrib"
-projects[wysiwyg][version] = "2.2"
+projects[wysiwyg][download][type] = "git"
+projects[wysiwyg][download][url] = "http://git.drupal.org/project/wysiwyg.git"
+projects[wysiwyg][download][revision] = "7981731f4f3db2f932419499d2ec13a073e9b88f"
```

Comment:
As I read it, the [module owner recommend using the dev version] (https://www.drupal.org/node/1853550#comment-9213657)